files/build_templates/docker_image_ctl.j2

```diff
@@ -209,39 +238,52 @@ start() {
         {%- endif %}
         if [ x"$DOCKERMOUNT" == x"$HWSKU" ]; then
             {%- if docker_container_name == "database" %}
-            echo "Starting existing {{docker_container_name}} container"
+            echo "Starting existing {{docker_container_name}}$DEV container"
             {%- else %}
-            echo "Starting existing {{docker_container_name}} container with HWSKU $HWSKU"
+            echo "Starting existing {{docker_container_name}}$DEV container with HWSKU $HWSKU"
             {%- endif %}
             preStartAction
-            docker start {{docker_container_name}}
-            CURRENT_HOSTNAME="$(docker exec {{docker_container_name}} hostname)"
+            docker start {{docker_container_name}}$DEV
+            postStartAction
+            CURRENT_HOSTNAME="$(docker exec {{docker_container_name}}$DEV hostname)"
             if [ x"$HOSTNAME" != x"$CURRENT_HOSTNAME" ]; then
                 updateHostName "$HOSTNAME"
             fi
-            postStartAction
+            updateHostName "$HOSTNAME"
             exit $?
         fi
 
         # docker created with a different HWSKU, remove and recreate
-        echo "Removing obsolete {{docker_container_name}} container with HWSKU $DOCKERMOUNT"
-        docker rm -f {{docker_container_name}}
+        echo "Removing obsolete {{docker_container_name}}$DEV container with HWSKU $DOCKERMOUNT"
+        docker rm -f {{docker_container_name}}$DEV
     fi
 
-{%- if docker_container_name == "database" %}
-    echo "Creating new {{docker_container_name}} container"
+    {%- if docker_container_name == "database" %}
+    echo "Creating new {{docker_container_name}}$DEV container"
     # if database_config exists in old_config, use it; otherwise use the default one in new image
     if [ -f /etc/sonic/old_config/database_config.json ]; then
         echo "Use database_config.json from old system..."
         mv /etc/sonic/old_config/database_config.json /etc/sonic/
     fi
-{%- else %}
-    echo "Creating new {{docker_container_name}} container with HWSKU $HWSKU"
-{%- endif %}
+    {%- else %}
+    echo "Creating new {{docker_container_name}}$DEV container with HWSKU $HWSKU"
+    {%- endif %}
+
+    if [ -z "$DEV" ]; then
+        NET="host"
+    else
+        {%- if docker_container_name == "database" %} 
+	NET="bridge"
```

Comment:
you are using tab, but other places you are using space in the same file. can you change to space and make the spacing consistent acrosss the file?

Reply:
Corrected spacing